Library/Homebrew/compat/macos.rb

```diff
@@ -90,5 +90,9 @@ def clang_version
     def clang_build_version
       DevelopmentTools.clang_build_version
     end
+
+    def has_apple_developer_tools?
+      DevelopmentTools.installed?
+    end
```

Comment:
It's not actually missing at this point. Looks like this should have been part of #455 instead.